.github/workflows/medic-benchmarks.yml

```diff
@@ -54,5 +54,17 @@ jobs:
           cluster: zeebe-cluster
           cluster-region: europe-west1-b
           ref: ${{ steps.data.outputs.ref }}
-          load: "worker, starter=100, timer=50, publisher=50"
+          load: "worker, starter=25, timer=25, publisher=25"
         secrets: inherit
+      - uses: ./.github/workflows/benchmark.yaml
+        name: Create latency benchmark
+        with:
+          name: medic-cw-${{ steps.data.outputs.cw }}-${{ steps.data.outputs.ref }}-benchmark-latency
+          cluster: zeebe-cluster
+          cluster-region: europe-west1-b
+          ref: ${{ steps.data.outputs.ref }}
+          load: "worker, starter=1"
+        secrets: inherit
+      - name: Downscale workers on latency benchmark
```

Comment:
:question: Here I was unsure whether the inherited resources/setup are here still available? Maybe you know that @oleschoenburg

Reply:
Here we don't have a working `kubectl` so the downscaling won't work. You could either:
* copy some of the setup steps from the benchmark workflow that setup gcloud and authenticate with the cluster
* support `worker = 1` syntax in the benchmark workflow to control the number of replicas.
* support arbitrary k8s json patches in the benchmark workflow. Here we could use it to patch the workers replica count.

Reply:
We migrated to https://github.com/camunda/zeebe/pull/11155 helm benchmarks :rocket: